src/resources/categories.js

```diff
@@ -0,0 +1,3 @@
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'].sort();
```

Comment:
[Optional] The sort method used here is redundant, as the array is already sorted. Also since the array is hard-coded, you can sort it by hand instead of using the sort method.

![image](https://user-images.githubusercontent.com/36057474/107001704-65713d80-678a-11eb-9fba-0726a7eed247.png)

Do you have a different reason to why you sorted this array? 🤔